### PR TITLE
feat: truncate text that doesn't fit and show tooltip with full text

### PR DESCRIPTION
**Pull request by mariobuikhuizen**
## Description

Long text in the table is pushing text of other cells out of view:
Screenshot 2021-09-28 at 12 24 48

With this PR the content is truncated and on hover a tooltip appears with the full text:
Screenshot 2021-09-28 at 12 27 01

### CODE REVIEW

File: glue_jupyter/table/table.vue [hunk 1]
```diff
@@ -63,8 +67,13 @@
 </template>
 
 
-<style scoped>
+<style id="glue_table">
```
Comment (maartenbreddels) [inline glue_jupyter/table/table.vue:70]: why this change? is this for the hot reloading?

**Review by maartenbreddels — APPROVED** [review]
Looks good, could you briefly describe the problem from a technical side as well as the solution?

File: glue_jupyter/table/table.vue (hunk 1 above)
Comment (mariobuikhuizen) [reply glue_jupyter/table/table.vue:70]: It's a small fix as `scoped` doesn't work in the ipyvue context and ideally all style tags should have an id to prevent duplicate style tags and leaking old styles during development.

It's not related to this PR, and shouldn't be here.

Comment (maartenbreddels) [reply glue_jupyter/table/table.vue:70]: ok, we can leave it in, but good to know it's not related.

**Comment by mariobuikhuizen** [comment]
I found the right way to fix this issue (using `table-layout: fixed;`)
